zio-sql/jvm/src/main/scala/Sql.scala

```diff
@@ -311,6 +312,11 @@ trait Sql {
         val _ = ev
         copy(groupBy = groupBy ++ (key :: keys.toList))
       }
+
+      def having(havingExpr2: Expr[_, A, Boolean])(
```

Comment:
Don't we need to assert on `F` (the feature) that everything is aggregated inside it, to conform to restrictions of having clause?

Reply:
Related issue: https://github.com/zio/zio-sql/issues/38